### PR TITLE
add scripts

### PR DESCRIPTION
**Pull request by Alexey-N-Chernyshov**

### CODE REVIEW

File: genesis.block [hunk 1]
```diff
@@ -45,7 +47,15 @@
                     "role_name": "voter",
                     "permissions": [
                         "can_transfer",
-                        "can_get_all_acc_ast"
+                        "can_get_all_acc_ast",
+                        "can_get_my_acc_ast_txs"
```
Comment (l4l) [inline genesis.block:59]: the previous perm should be enough

File: example.py [hunk 2]
```diff
@@ -0,0 +1,264 @@
+import sys
+sys.path.insert(0, 'build/shared_model/bindings')
```
Comment (l4l) [inline example.py]: better remove and use env variables

File: example.py [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+
+key_pair = crypto.convertFromExisting(manager_pub, manager_priv)
+
+current_time = int(round(time.time() * 1000)) - 10**5
+creator = "manager@global"
+tx_counter = 1
+query_counter = 1
+
+def get_time():
+    return int(round(time.time() * 1000)) - 10**5
```
Comment (l4l) [inline example.py]: why `- 10**5`?

File: example.py [hunk 4]
```diff
@@ -0,0 +1,264 @@
...
+crypto = iroha.ModelCrypto()
+proto_tx_helper = iroha.ModelProtoTransaction()
+proto_query_helper = iroha.ModelProtoQuery()
+
+manager_pub = open("manager@global.pub", "r").read()
+manager_priv = open("manager@global.priv", "r").read()
+
+key_pair = crypto.convertFromExisting(manager_pub, manager_priv)
+
+current_time = int(round(time.time() * 1000)) - 10**5
```
Comment (l4l) [inline example.py]: there's a function below for that

File: example.py [hunk 5]
```diff
@@ -0,0 +1,264 @@
...
+def get_time():
+    return int(round(time.time() * 1000)) - 10**5
+
+def get_status(tx):
+    # Create status request
+
+    print("Hash of the transaction: ", tx.hash().hex())
+    tx_hash = tx.hash().blob()
+
+    if sys.version_info[0] == 2:
```
Comment (l4l) [inline example.py]: move that to fn plz

File: example.py [hunk 6]
```diff
@@ -0,0 +1,264 @@
...
+    print_status_streaming(tx)
+
+def add_candidate(candidate, domain, tx_buidler):
+    return tx_buidler \
+        .createAccount(candidate.name, domain, candidate.keypair.publicKey()) \
+        .appendRole(candidate.name + '@' + domain, 'candidate')
+
+def add_voter(voter, domain, tx_buidler):
+    return tx_buidler \
+        .appendRole(voter.name + '@global', 'prevote') \
```
Comment (l4l) [inline example.py]: why?

File: genesis.block (hunk 1 above)
Comment (Alexey-N-Chernyshov) [reply genesis.block:59]: I want voter to see only her own transactions (to check it was correct). I think can_get_all_acc_ast should be removed

File: example.py (hunk 3 above)
Comment (Alexey-N-Chernyshov) [reply example.py]: I dunno, got it from examples

File: example.py (hunk 6 above)
Comment (Alexey-N-Chernyshov) [reply example.py]: to add asset (vote) for account and then prohibit transfer inter accounts.

**Pull request closed without merging**